bblocks/import_tools/imf.py

```diff
@@ -192,12 +192,11 @@ def load_indicator(self, indicator: str = "all") -> ImportData:
 
         if (
             not os.path.exists(f"{PATHS.imported_data}/{self.file_name}")
-            or self.update_data
+            or (self.update_data and self.data is None)
```

Comment:
why the extra `self.data is None` check?

Reply:
This it to prevent the data from being downloaded every time `load_indicator` is called (if update_data is True). It will only update the data on the first call i.e when self.data is None. Is there a better way to do this?

Reply:
aha no i get it. Sounds good!